Sources/SwiftGenStringsCore/StandardErrorOutput.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public class StandardErrorOutput {
+
+    private var writtenErrors = [Error]()
```

Comment:
writtenErrorMessage would probably be more easy to consume, since we are not adding any more information with the NSError instances